src/js/components/Chart/doc.js

```diff
@@ -58,6 +58,9 @@ export const doc = Chart => {
     id: PropTypes.string.description(`A unique identifier for the Chart. This
       is required if more than one Chart is shown and they use color
       gradients.`),
+    dash: PropTypes.oneOfType([PropTypes.bool, PropTypes.string])
+      .description(`Whether to use dashed lines for line or bar charts.`)
```

Comment:
This description is clear for the boolean value, but not for the string value.
I'm thinking we should add that the string will be used as a CSS `stroke-dasharray`, otherwise, it is not clear what are the string values that are possible for `dash`.

Reply:
Simplified to remove the string capability for now.